src/DogProfile.js

```diff
@@ -1,9 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './DogProfile.css'
+
+const DogProfile = ({ id, name, age, breed }) => {
+    
+    const [ skills, setSkills ] = useState([])
 
-const DogProfile = ({ id }) => {
     return(
         <div>
-            <h1>{ id }</h1>
+            <div className='dog-profile-container'>
```

Comment:
This dog profile styling looks great. Now we just need to hook up the skills and map through them.